<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/logo.png</normaloff>:/icons/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow {background-color: rgb(255, 255, 255);}</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuShape">
    <property name="title">
     <string>Shape</string>
    </property>
    <addaction name="actionLine"/>
    <addaction name="actionCircle"/>
    <addaction name="actionRectangle"/>
    <addaction name="actionEllipse"/>
    <addaction name="actionArc"/>
    <addaction name="actionFreehand"/>
   </widget>
   <widget class="QMenu" name="menuCOlor">
    <property name="title">
     <string>Color</string>
    </property>
    <addaction name="actionBlack"/>
    <addaction name="actionRed"/>
    <addaction name="actionGreen"/>
    <addaction name="actionBlue"/>
    <addaction name="actionChoose_Color"/>
   </widget>
   <widget class="QMenu" name="menuLine_Style">
    <property name="title">
     <string>Line Style</string>
    </property>
    <addaction name="actionSolid"/>
    <addaction name="actionDotted"/>
    <addaction name="actionDashDot"/>
    <addaction name="actionDashDotDot"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionNormal"/>
    <addaction name="actionFilled"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClear_Drawings"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuShape"/>
   <addaction name="menuCOlor"/>
   <addaction name="menuLine_Style"/>
   <addaction name="menuMode"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLine"/>
   <addaction name="actionCircle"/>
   <addaction name="actionRectangle"/>
   <addaction name="actionEllipse"/>
   <addaction name="actionFreehand"/>
   <addaction name="actionArc"/>
  </widget>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/line.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/circle.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/rectangle.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/ellipse.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Ellipse</string>
   </property>
  </action>
  <action name="actionArc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/arc.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Arc</string>
   </property>
  </action>
  <action name="actionFreehand">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/freehand.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Freehand</string>
   </property>
  </action>
  <action name="actionRed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </action>
  <action name="actionGreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
  </action>
  <action name="actionBlue">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </action>
  <action name="actionSolid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/solid.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Solid</string>
   </property>
  </action>
  <action name="actionDotted">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/dotted.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Dotted</string>
   </property>
  </action>
  <action name="actionBlack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Black</string>
   </property>
  </action>
  <action name="actionDashDot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/dashdot.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>DashDot</string>
   </property>
  </action>
  <action name="actionDashDotDot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/dashdotdotdah.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>DashDotDot</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="actionFilled">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Filled</string>
   </property>
  </action>
  <action name="actionChoose_Color">
   <property name="text">
    <string>Choose Color</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClear_Drawings">
   <property name="text">
    <string>Clear Drawings</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
